src/OTSubscriberView.js

```diff
@@ -42,22 +43,25 @@ export default class OTSubscriberView extends React.Component {
       audioVolume,
       preferredFrameRate,
       preferredResolution,
-      subscribeToAudio,
       subscribeToCaptions,
-      subscribeToVideo,
-      style,
     } = subscriberProperties;
     if (streamProperties) {
       ({
         audioVolume,
         preferredFrameRate,
         preferredResolution,
-        subscribeToAudio,
         subscribeToCaptions,
-        subscribeToVideo,
-        style,
       } = streamProperties);
     }
+    const subscribeToVideo =
+      streamProperties?.subscribeToVideo ??
+      subscriberProperties?.subscribeToVideo ??
+      true;
```

Comment:
Note that this uses the JavaScript optional chaining (`?`) is and nullish coalescing (`??`) operators. In short, the `streamProperties.subscribeToVideo` takes precidence over the `subscriberProperties`.subscribeToVideo` properties. But if they are both unset, the expression returns `true` (the default for `subscribeToVideo`).